app/mailers/user_interface_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class UserInterfaceMailer < ActionMailer::Base
+  default from: ENV['DEFAULT_EMAIL_FROM_CARDS']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.